### DISCUSSION TITLE
Keycloak Get SMTP Password with API

### DISCUSSION THREAD

**Discussion by jonathan-fortin** [keycloak/keycloak#15839] [Q&A]
Hi,

I want to use Keycloak SMTP configuration in my app to send specific emails. But when I retrieve the configuration using the API, the SMTP authentication password is hidden. It is replaced by stars in the json response.

Is there a way to get the value with the API?

Thanks

**Comment by stianst** [comment]
Nopes

**Comment by jonathan-fortin** [comment]
Hi,

If it is not possible. Is it possible to send emails using the Keycloak API? Not to validate the user, but to send any type of email using the config?

Regards,
